src/Microsoft.AspNetCore.DataProtection.Redis/RedisDataProtectionBuilderExtensions.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using StackExchange.Redis;
+using Microsoft.AspNetCore.DataProtection.Repositories;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.DataProtection
+{
+    /// <summary>
+    /// Contains Redis-specific extension methods for modifying a <see cref="IDataProtectionBuilder"/>.
+    /// </summary>
+    public static class RedisDataProtectionBuilderExtensions
+    {
+        private const string DataProtectionKeysName = "DataProtection-Keys";
```

Comment:
Should we even have this default key (and the extensions using it) and always force users to supply a key? Since a single Redis server could be used by different applications, having this same key could be problematic. (or is the idea that one could supply different databases for different apps?)...I do not know much about Redis so not sure if my assumptions are right...